Makefile

```diff
@@ -200,7 +205,7 @@ endif
 # Generate bundle manifests and metadata, then validate generated files.
 bundle: kustomize operator-sdk manifests
 	$(OPERATOR_SDK) generate kustomize manifests -q
-	$(KUSTOMIZE) build config/manifests | $(OPERATOR_SDK) generate bundle -q --overwrite --manifests --version $(VERSION) $(BUNDLE_METADATA_OPTS)
+	$(KUSTOMIZE) build config/manifests | $(OPERATOR_SDK) generate bundle -q --overwrite --manifests --version $(OPERATOR_VERSION) $(BUNDLE_METADATA_OPTS)
```

Comment:
By default, the version here is something like `0.29.0-5-gf6a55f6` but for CI level validation we need the current operator version. So adding a new variable with the name `OPERATOR_VERSION` which picks operator version from `versions.txt` 

I am not sure of any other issues with generating bundle files with the current version, But open for suggestions here @jpkrohling

Reply:
I'm not quite sure. I think I would prefer to have `VERSION=x.y.z make bundle` (or `ensure-...`) as part of the CI instead. Generating a dev bundle (`make bundle` locally) should still use the notation we are currently using.